### DISCUSSION TITLE
Don't ignore trailing zeros in buzzer tune

### DISCUSSION THREAD

**Discussion by hazymat** [arendst/Tasmota#10694] [Ideas]
By design, when creating buzzer "tunes", Tasmota ignores leading AND trailing zeros.

It would be sensible to ignore leading zeros but NOT trailing zeros, as this gives the ability to create a "tune" that repeats, without losing the silences at the end of each 32 bit mask.

For example, imagine this one on repeat:
11000011001100101000100010000000

If the above is repeated, the last zeros allow this repeating rhythm to sound really good, but if they get cut off, it doesn't make sense musically.

By ignoring ONLY leading zeros, this still means it's possible not to "waste CPU cycles", but also does allow the above.

**Comment by arendst** [comment]
Just added in development version
